dashboard/src/components/ClustersList.tsx

```diff
@@ -133,7 +133,6 @@ function ClustersList({
               onChange={(event) => {
                 onNamespaceChange(event.target.value);
               }}
-              checked={filteredNamespaces?.includes(namespace.name)}
```

Comment:
did you made sure all of the functionality regarding namespaces works as expected?
install chart, delete chart, upgrade, downgrade, etc.

Reply:
yes